storage/replica.go

```diff
@@ -174,10 +173,12 @@ type Replica struct {
 	tsCache      *TimestampCache // Most recent timestamps for keys / key ranges
 	pendingSeq   uint64          // atomic sequence counter for cmdIDKey generation
 	pendingCmds  map[cmdIDKey]*pendingCmd
+	desc         *roachpb.RangeDescriptor
 
 	truncatedState unsafe.Pointer // *roachpb.RaftTruncatedState
 
 	replicaID roachpb.ReplicaID
+	RangeID   roachpb.RangeID // Should only be set by the constructor.
```

Comment:
Put this at the top of the struct; fields declared here are under the `RWMutex`.